src/source/Ice/TurnConnection.h

```diff
@@ -37,6 +37,8 @@ extern "C" {
 #define DEFAULT_TURN_CHANNEL_DATA_BUFFER_SIZE             512
 #define DEFAULT_TURN_MAX_PEER_COUNT                       32
 
+#define DEFAULT_TURN_ALLOCATION_MAX_TRY_COUNT             3 
```

Comment:
I wonder if folks from AWS can comment on the choice of the default max retry count being 3. It seems to be a good candidate.